integration-tests/lib/commands.py

```
@@ -0,0 +1,134 @@
+
+    return (
+        Program("python3"),
+        Program("podman"),
+        Program("git"),
+        Program("gh"),
+        Program("coderabbit"),
+        Program("sh"),
+    )
+
+
+def host_catalogue() -> "ProgramCatalogue":  # type: ignore[name-defined]
```

Ruff (0.15.14)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)